File: SOLID/app/src/main/java/food/items/MenuItem.java
package food.items;

public abstract class MenuItem {
    protected String name;
    protected String description;
    protected double price;

    public MenuItem(String name, String description, double price) {
        this.name = name;
        this.description = description;
        this.price = price;
    }

    public String getDescription() {
        return this.description;
    }

    public String getName() {
        return this.name;
    }

    public double getPrice() {
        return this.price;
    }
}

File: StructuralPatterns/menu/MenuItem.py
class MenuItem():
    def __init__(self, name: str = None, price: float = None) -> None:
        self._name = name
        self._price = price

    def show(self):
        return self._name

    def get_price(self):
        return self._price

    def get_name(self):
        return self._name

    def __repr__(self) -> str:
        return f"{self._name} - {self._price}"
